.github/workflows/build-deploy-feature-branch-q1.yaml

```diff
@@ -5,6 +5,8 @@ env:
   IMAGE_TAG: ${{ github.sha }}
   IMAGE: ghcr.io/${{ github.repository }}/veilarbportefolje
   PRINT_PAYLOAD: true
+permissions:
+  packages: write
 
```

Comment:
Kan du raskt forklare hvorfor dette trengs? :)

Reply:
https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#about-scopes-and-permissions-for-package-registries

Reply:
Fant også denne posten her: https://nav-it.slack.com/archives/D03UK0VHAF5/p1693825855373549